modules/ROOT/pages/export-to-pdf-with-jwt-authentication-nodejs.adoc

```diff
@@ -0,0 +1,215 @@
+
+This guide provides a comprehensive walkthrough for integrating {pluginname} with {productname}, including {pluginname} functionality, by using a Node.js server for JWT token generation. It covers project setup, server configuration, and {productname} customization.
+
+== What You'll Build
+
+Before diving into the technical details, here's what you'll achieve with this guide:
+
+* A working PDF export system for your TinyMCE editor
+* A secure authentication system using JWT tokens
+* A simple Node.js server to handle the authentication
+
+[TIP]
+====
+This guide is designed for developers new to JWT authentication and TinyMCE integration.
+====
+
+== Prerequisites
+
+Before starting, ensure you have:
+
+* Node.js installed on your computer
+* A TinyMCE API key (get one from link:https://www.tiny.cloud/signup[TinyMCE's website])
+* Basic familiarity with the command line
+
+[IMPORTANT]
+====
+Make sure you have your API key ready before starting. You'll need it for both the server and client configuration.
+====
+
+== Quick Start Guide
+
+=== Project Setup (5 minutes)
+
+[source,bash]
+----
+# Create and enter project directory
+mkdir tinymce-pdf-export
+cd tinymce-pdf-export
+
+# Initialize project
+npm init -y
+
+# Install required packages
+npm install express cors jsonwebtoken
+----
+
+Verify that the `package.json` file includes the below required dependencies:
+
+[source,json]
+----
+{
+  "dependencies": {
+    "cors": "^2.8.5",
+    "express": "^4.21.0",
+    "jsonwebtoken": "^9.0.2"
+  }
+}
+----
+
+=== Create Project Structure
+
+[source,bash]
+----
+# Create the public folder for your web files
+mkdir public
+touch public/index.html
+touch jwt.js
+----
+
+Your project should look like this:
+
+[source]
+----
+/tinymce-pdf-export
+  /public
+    index.html  (TinyMCE webpage)
+  jwt.js       (Server code)
+  package.json (Project configuration)
+----
+
+. Inside the `public` folder where you created the `index.html` file add the HTML setup code (refer to the *Setting up {productname} HTML* section).
+
+==== Web Page Setup (public/index.html)
+
+[source,html]
+----
+<!DOCTYPE html>
+<html>
+<head>
+  <title>TinyMCE with PDF Export</title>
+  <script 
+    src="https://cdn.tiny.cloud/1/YOUR-API-KEY/tinymce/7/tinymce.min.js"
+    referrerpolicy="origin">
+  </script>
+  <script>
+    tinymce.init({
+      selector: 'textarea',
+      plugins: 'exportpdf',
+      toolbar: 'exportpdf',
+      exportpdf_converter_options: {
+        'format': 'Letter',
+        'margin_top': '1in',
+        'margin_right': '1in',
+        'margin_bottom': '1in',
+        'margin_left': '1in'
+      },
+      exportpdf_service_url: "<serviceURL>",
```

Comment:
This is a required option for paid service, but is not required when testing on trial account @LewisAtTiny 

```suggestion
      exportpdf_service_url: "<serviceURL>",
```